spring-batch-bigquery/pom.xml

```diff
@@ -106,39 +89,42 @@
             <version>${logback.version}</version>
```

Comment:
If we switch to spring boot parent then this version can be automatically taken from the parent instead of specifying it manually.

Reply:
Indeed, it was kind of the purpose of this PR, and I missed it 😅 thanks for catching it